packages/cli/src/applyPreset.ts

```diff
@@ -44,6 +47,9 @@ else if (network) {
 else if (process.argv[2] === "dev") {
   process.env.LODESTAR_PRESET = "minimal";
   process.env.LODESTAR_NETWORK = "dev";
+  // "c-kzg" has hardcoded the mainnet value, do not use presets
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  setActivePreset(PresetName.minimal, {FIELD_ELEMENTS_PER_BLOB: 4096});
```

Comment:
Adding this in the production code path just for a local test of us is pretty ugly. Please use presetFile functionality in that test to override the values on the test runtime only.

Reply:
this for running lodestar via `--dev` network flag (mostly via prebuild/docker images), test override is a different file

Reply:
Could you detail more who / where this is necessary? Is someone else dependant on this fix?

Reply:
generally its used in ethereumjs sims for 4844, also we would want our `--dev` network to be 4844 compatible with EL clients else we should  drop `--dev` functionality.